static/locales/pt-PT.yaml

```diff
@@ -182,15 +182,15 @@ Settings:
     Save Watched Progress: Guardar onde ficou da última vez num vídeo
     Clear Search Cache: Limpar Cache de Pesquisas
     Are you sure you want to clear out your search cache?: Tem a certeza de que quer
-      apagar o seu cache de pesquisas?
```

Comment:
This change in particular is what makes me question him, as it was originally correct, but was changed to be incorrect.